### DISCUSSION TITLE
ESP32 Echo voice only second time

### DISCUSSION THREAD

**Discussion by imagica71** [vintlabs/fauxmoESP#169] [Q&A]
Hello,
i got my test running with the basic example.
However, when the fauxmoESP project is running for a while (maybe 3-5 mins or so), the voice command is not recognized anymore at the first time (echo says "device not responding") but if i speak the command again the second time it does. As said, this happens only after a while.

Any suggestions?

**Comment by Tejas-MD** [comment]
I'm facing the same issue since recently and am trying to debug it. Will update if I can fix.

**Comment by Tejas-MD** [comment]
I noticed that it was working fine the day it was added and got worse progressively as time passes. Hence I tried discovering devices more frequently but this didn't seem to help.

`And I'm trying to add ``fauxmo.handle()``  multiple times in the loop just in case it's not being called on time. (Testing this in progress, good working so far: 1 week) 


Also, I'm starting to think it is related to [Discussion #184](https://github.com/vintlabs/fauxmoESP/discussions/184)

@pvint please suggest your advice on this or #184, pretty annoying hence urgent.
